src/CapitalInvestments/PyomoModels/ModelBase.py

```diff
@@ -5,49 +5,26 @@
   @author: wangc, mandd
 """
```

Comment:
This changes are already present in PR #28, no?

Reply:
I'm sorry I forget to mention. This PR is based on PR#28. PR#28 should be merged first before this PR.